components/logins/build.rs

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+fn main() {
+    println!("cargo:rerun-if-changed=src/places_msg_types.proto");
```

Comment:
This file is wrong. You should be adding a new message type file to `src/logins_msg_types.proto` (unfortunately the `logins_` name prefix is required due to some idiosyncracies in the iOS build process), and not referring to the places types and etc.

Reply:
Thanks, I have seen the comment on top of the `.proto` file.